### ISSUE TITLE
Doc Settings panel wraps wrongly in some laguages

### ISSUE THREAD

**Issue by fflorent** [gristlabs/grist-core#1029]
Here is a preview of the Doc settings panel in French, it's not very pretty :)

![image](https://github.com/gristlabs/grist-core/assets/371705/68e2d5ad-8883-4400-a246-59ac9c67c9cc)

A solution we could implement would be to wrap the second column (the one with the description) so it does not take all the place to the right, and also to ensure all the three columns remains on the same line.

**Comment by dsagal** [comment]
Definitely, forcing the same line, and allowing text to wrap would be better. It might be OK to wrap text inside buttons too, or just to use a shorter label (since there is already a description).

**Comment by hexaltation** [comment]
Fixed in a previous DocSetting page revamp #1376 
Please feel free to reopen if needed.
